react/Reviews.tsx

```diff
@@ -598,25 +604,31 @@ function Reviews() {
     })
   }, [dataAverage, loadingAverage, state.total, state.reviews])
 
+  useEffect(() => {
```

Comment:
Why not combine this with the `useEffect` above? They both have the same dependencies, so you could handle everything at the same time